### PR TITLE
(WIP) (IMAGES-7832) Add Win-2008r2 base template 

### PR DESCRIPTION
**Pull request by jcoconnor**
Add definitions for Windows 2008R2 base image for Packer/Windows Imaging.
Further PR's will follow to handle the 2008 compatibility issues (i.e.
PS2) and the cygwin updates.

### CODE REVIEW

**Comment by jcoconnor** [comment]
This has been verified as building on the Jenkins Imaging machine.

Further validation work will follow with later PR's to get the Cygwin and other Win-2008R2 specific work done.

File: templates/windows-2008r2/files/autounattend.xml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                            <Order>1</Order>
+                            <PartitionID>1</PartitionID>
+                            <Label>System</Label>
+                            <Format>FAT32</Format>
+                        </ModifyPartition>
+                        <!-- Windows partition -->
+                        <ModifyPartition wcm:action="add">
+                            <Order>2</Order>
+                            <PartitionID>3</PartitionID>
+                            <Label>Windows w2008R2</Label>
```
Comment (glennsarti) [inline templates/windows-2008r2/files/autounattend.xml]: Typically the volume name is just WINDOWS...Any particular reason to modify it?

**Comment by jcoconnor** [comment]
This PR is being included in #140

File: templates/windows-2008r2/files/autounattend.xml (hunk 1 above)
Comment (jcoconnor) [inline templates/windows-2008r2/files/autounattend.xml]: Just a preference on my part.

**Pull request closed without merging**